types/craco__craco/package.json

```
@@ -0,0 +1,7 @@
+{
+    "private": true,
+    "dependencies": {
+        "@jest/types": "^27.4.2",
+        "@types/autoprefixer": "^9.7.2"
```

```suggestion
        "autoprefixer": "^10.4",
        "@jest/types": "^27.4.2"
```
`autoprefixer` is only used to import options type for reuse, can we use native types if possible? (unless there is a reason to use v9)

---

@craco/craco is using autoprefixer v9, I'm afraid there may be incompatibility between major versions